test/pymongo_mocks.py

```diff
@@ -141,13 +142,32 @@ def __init__(
         self.mock_rtts = {}
 
         kwargs["_pool_class"] = partial(MockPool, self)
-        kwargs["_monitor_class"] = partial(MockMonitor, self)
+        kwargs["_monitor_class"] = partial(SyncMockMonitor, self)
 
         client_options = client_context.default_client_options.copy()
         client_options.update(kwargs)
 
         super().__init__(*args, **client_options)
 
+    @classmethod
+    def get_async_mock_client(
```

Comment:
Should this be `get_sync_mock_client`?

Reply:
Good catch, using `get_mock_client` seems better.